src/test/java/api/loans/ClaimItemReturnedAPITests.java

```diff
@@ -0,0 +1,103 @@
+package api.loans;
+
+import static api.support.http.InterfaceUrls.claimItemReturnedURL;
+import static api.support.matchers.LoanMatchers.hasLoanProperty;
+import static api.support.matchers.LoanMatchers.hasOpenStatus;
+import static api.support.matchers.LoanMatchers.hasStatus;
+import static api.support.matchers.ValidationErrorMatchers.hasErrorWith;
+import static api.support.matchers.ValidationErrorMatchers.hasMessage;
+import static api.support.matchers.ValidationErrorMatchers.hasParameter;
+import static org.folio.circulation.domain.ClaimItemReturnedRequest.ITEM_CLAIMED_RETURNED_DATE;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION_COMMENT;
+import static org.folio.circulation.domain.representations.LoanProperties.CLAIMED_RETURNED_DATE;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class ClaimItemReturnedAPITests extends APITests {
+  private InventoryItemResource item;
+  private UUID loanId;
+
+  @Before
+  public void setUpItemAndLoan() {
+    item = itemsFixture.basedUponSmallAngryPlanet();
+    loanId = loansFixture.checkOutByBarcode(item, usersFixture.charlotte()).getId();
+  }
+
+  @Test
+  public void canMakeItemClaimedReturnedWithComment() {
+    final String comment = "testing";
+    final DateTime dateTime = DateTime.now();
```

Comment:
How about using `ClockManager.getClockManager().getDateTime();`  instead of `DateTime.now();` ?

Reply:
It is not required for unit tests, but for production code only.